tests/report.py

```diff
@@ -0,0 +1,18 @@
+import unittest
+import easypost
+from constants import API_KEY as api_key
+
+easypost.api_key = api_key
+
+class ReportTests(unittest.TestCase):
+
+  def test_report_creation(self):
```

Comment:
This test does not pass; looks to be using Parcel instead of Report for some reason